apps/rewards/app/components/Card/Empty.js

```diff
@@ -4,7 +4,7 @@ import styled from 'styled-components'
 import { Button, EmptyStateCard, GU, LoadingRing, unselectable } from '@aragon/ui'
 import icon from '../../assets/empty-rewards.svg'
 
-const Icon = () => <img src={icon} height={20 * GU} />
+const Icon = () => <img src={icon} height={20 * GU} alt="No rewards" />
```

Comment:
I doubt we want this – looking below, we already say "No rewards here!" Having the alt text repeat that message will result in screen readers saying "No rewards no rewards here!"

Instead, let's set it to empty. `alt=""`